src/services/tasks/tasks.ts

```diff
@@ -38,4 +39,5 @@ export default (client: HttpClient, httpAuth: HttpInstance): TasksService => ({
   },
 
   api: api(client, httpAuth),
+  logs: logs(client, httpAuth),
```

Comment:
For context I hate this interface is called `logs` but it resolves to exh.tasks.logs, which to me is much nicer than exh.tasks.taskLogs